models/FeedCategory.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+module.exports = function(sequelize, DataTypes) {
+  var FeedCategory = sequelize.define('FeedCategory', {
+    id: {
+      type: DataTypes.INTEGER(11),
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    owner_uid: {
+      type: DataTypes.INTEGER(11),
+      allowNull: false
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    collapsed: {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+      defaultValue: '0'
+    },
+    order_id: {
+      type: DataTypes.INTEGER(11),
+      allowNull: false,
+      defaultValue: '0'
+    },
+    parent_cat: {
+      type: DataTypes.INTEGER(11),
+      allowNull: true
+    },
+    view_settings: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      defaultValue: ''
+    }
+  }, {
+
+    //
+    // Options
+    //
+
+    name: {
+      plural: 'FeedCategories',
+      singular: 'FeedCategory'
+    },
+    tableName: 'ttrss_feed_categories',
+
+
+    //
+    // static methods
+    //
+
+    classMethods: {
+
+      /**
+       * method used to declare all model associations
+       * @param {Object} models
+       */
+      associate: models => {
```

Comment:
Let's keep this method out of models and make associates in `models/index.js` for now. I tried it this way before and am not sure yet what the best approach is.